2.4/root/usr/share/container-scripts/mongodb/common.sh

```diff
@@ -103,10 +97,12 @@ function build_mongo_config() {
   member_id=1
   local container_addr
   container_addr="$(container_addr)"
+  local node
   for node in ${current_endpoints}; do
-    [ "$node" == container_addr ] && continue
-    members+="{ _id: ${member_id}, host: \"${node}:${CONTAINER_PORT}\"},"
-    let member_id++
+    if [ "$node" != "$container_addr" ]; then
```

Comment:
> I believe it is not. And I believe it doesn't matter here. When we initialize the replica set the pod running the hook is the PRIMARY, and later it steps down.

And this pod runs the initialization. So the compating with container_addr is redundant - post-hook-pod IP (=container_addr) won't be in current_endpoints...